pkg/servechi/servechi.go

```diff
@@ -0,0 +1,197 @@
+package servechi
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/Contrast-Security-OSS/go-test-bench/internal/common"
+	"github.com/Contrast-Security-OSS/go-test-bench/internal/injection/cmdi"
+	"github.com/Contrast-Security-OSS/go-test-bench/internal/injection/sqli"
+	"github.com/Contrast-Security-OSS/go-test-bench/internal/pathtraversal"
+	"github.com/Contrast-Security-OSS/go-test-bench/internal/ssrf"
+	"github.com/Contrast-Security-OSS/go-test-bench/internal/unvalidated"
+	"github.com/Contrast-Security-OSS/go-test-bench/internal/xss"
+	"github.com/go-chi/chi"
+)
+
+// Pd is unchanging parameter data shared between all routes.
+var Pd = common.ConstParams{
+	Year:      2022,
+	Logo:      "https://blog.golang.org/gopher/header.jpg",
```

Comment:
It'd be nice if we could use the chi logo: https://cdn.rawgit.com/go-chi/chi/master/_examples/chi.svg